docs: Refresh docs for SQL input source (#17031)
apache/druid: docs/development/extensions-core/druid-lookups.md

:::info

>  If using JDBC, you will need to add your database's client JAR files to the extension's directory.

Reviewer: ```suggestion
To use JDBC, you must add your database client JAR files to the extension's directory.
```
nit: avoid future tense
